Containerfile.acm.konflux

```diff
@@ -0,0 +1,39 @@
+FROM registry.redhat.io/ubi8/nodejs-18:1-40 AS builder
+
+USER root
+ENV NPM_CONFIG_NODEDIR=/usr
+
+WORKDIR /app
+COPY . .
+
+# Running installs concurrently fails on aarch64
+RUN npm ci --no-optional --unsafe-perm --ignore-scripts
+RUN cd backend && npm ci --no-optional  --unsafe-perm
+RUN cd frontend && npm ci --no-optional --legacy-peer-deps --unsafe-perm
+RUN npm run build:backend
+RUN cd frontend && npm run build:plugin:acm
+
+# Remove build-time dependencies before packaging
+RUN cd backend && npm ci --no-optional --only=production --unsafe-perm
+
+FROM brew.registry.redhat.io/rh-osbs/rhacm2-nodejs-parent:v2.9.0_18-2
+
+WORKDIR /app
+ENV NODE_ENV production
+COPY --from=builder /app/backend/node_modules ./node_modules
+COPY --from=builder /app/backend/backend.mjs ./
+COPY --from=builder /app/frontend/plugins/acm/dist ./public/plugin
+USER 1001
+CMD ["node", "backend.mjs"]
+
+LABEL com.redhat.component="console-container" \
+      name="rhacm2/console-rhel9" \
```

Comment:
Name is inaccurate here - this image file specifies ubi8, not ubi9. Either update the image version to match what is downstream, or change the name and we'll update versions for both MCE and ACM in a second ticket.